mcs/class/System/Test/System.Net.Sockets/UdpClientTest.cs

```diff
@@ -558,25 +558,12 @@ public void JoinMulticastGroup1_Socket_Closed ()
 		}
 
 		[Test] // JoinMulticastGroup (IPAddress)
-		[Category ("NotWorking")]
 		public void JoinMulticastGroup1_Socket_NotBound ()
 		{
 			IPAddress mcast_addr = IPAddress.Parse ("224.0.0.23");
 
-			UdpClient client = new UdpClient (AddressFamily.InterNetwork);
-			try {
+			using (UdpClient client = new UdpClient (AddressFamily.InterNetwork)) {
 				client.JoinMulticastGroup (mcast_addr);
-				Assert.Fail ("#1");
```

Comment:
Why are we no longer expecting an exception here? Was the test always wrong before?